test/e2e/reconfigure_test.go

```diff
@@ -23,10 +23,6 @@ func reconfigurationTestSuite(t *testing.T) {
 	if tc.CloudProvider.GetType() == config.VSpherePlatformType {
 		t.Run("Re-add removed instance", tc.testReAddInstance)
 	}
-	// testPrivateKeyChange must be the last test run in the reconfiguration suite. This is because we do not currently
```

Comment:
Isn't this still the case? I am not sure we can move this test around without things going wrong.